lib/nghttp3_ringbuf.c

```diff
@@ -34,20 +34,9 @@
 #include "nghttp3_macro.h"
 
 #ifndef NDEBUG
-static int ispow2(size_t n) {
-#  if defined(DISABLE_POPCNT) ||                                               \
-    (defined(_MSC_VER) && !defined(__clang__) &&                               \
-     (defined(_M_ARM) || (defined(_M_ARM64) && _MSC_VER < 1941)))
-  return n && !(n & (n - 1));
-#  elif defined(WIN32)
-  return 1 == __popcnt((unsigned int)n);
-#  else  /* !((defined(_MSC_VER) && !defined(__clang__) && (defined(_M_ARM) || \
-            (defined(_M_ARM64) && _MSC_VER < 1941))) || defined(WIN32)) */
-  return 1 == __builtin_popcount((unsigned int)n);
-#  endif /* !((defined(_MSC_VER) && !defined(__clang__) && (defined(_M_ARM) || \
-            (defined(_M_ARM64) && _MSC_VER < 1941))) || defined(WIN32)) */
-}
-#endif /* !defined(NDEBUG) */
+/* Power-of-two test; simple portable bit trick. */
+static int ispow2(size_t n) { return n && !(n & (n - 1)); }
+#endif /* !NDEBUG */
```

Comment:
```suggestion
#endif /* !defined(NDEBUG) */
```
This is what we do consistently in this codebase as the original code did.

Reply:
Fixed by reverting back